install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1772,7 +1772,11 @@ loadBalancer:
   # mode: snat
 
   # -- acceleration is the option to accelerate service handling via XDP
-  # e.g. native, disabled
+  # disabled - do not use XDP
+  # native - XDP BPF program is run directly out of the networking driver’s
```

Comment:
Please avoid curly apostrophes, they're a pain for grepping.

Reply:
Oh weird. Not sure how that snuck in.

Reply:
Not sure in this particular case, but I sometimes see them sneaking in when someone copy-paste from a Google doc or some other source that turned straight apostrophes/quotes to curly ones automatically.